scripts/Engineering/gui/engineering_diffraction/tabs/fitting/data_handling/data_model.py

```diff
@@ -164,7 +164,9 @@ def get_ws_sorted_by_primary_log(self):
             log_table = ADS.retrieve(primary_log)
             isort = argsort(array(log_table.column('avg')))
             ws_list_tof = [ws_list[iws] for iws in isort if iws in tof_ws_inds]
-        if not sort_ascending == 'true':
+        else:
+            ws_list_tof = ws_list
+        if sort_ascending == 'false':
```

Comment:
I think there may have been a reason it was `not "true"` in the settings it might be false is actually denoted by an empty string (something to do with how the settings are saved and loaded)- but that might have been resolved at a later date. Can you just confirm that the value is "false" when sorting in descending order?

Reply:
I've just tried it in the debugger and it seems to return `false` correctly when specified in the settings